exercise-1/script.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('button');
+const nav = document.querySelector('nav');
+const main = document.querySelector('.hide-main');
+
+button.addEventListener('click', function () {
+  nav.classList.toggle('display');
+  main.classList.toggle('hide-main');
+});
+
+function keyPress(e) {
+  if (e.key === 'Escape') {
+    console.log('hello');
+  } else false;
+}
```

Comment:
This looks like dead code (unused function). I think it's safe to just remove it.